.github/workflows/tests.yml

```diff
@@ -90,30 +117,51 @@ jobs:
         env:
           # https://github.com/shivammathur/setup-php/issues/407#issuecomment-773675741
           fail-fast: true
-
+      - name: Check phpinfo()
+        run: php -i
+      - name: Check Composer Version
+        run: composer -V
+      - name: Check PHP Extensions
+        run: php -m
```

Comment:
I don't think all of this is useful to have it. You may check which php version is used and what extensions are enabled in each action run when you expand the "Setup PHP" section